plugins/experimental/access_control/unit_tests/test_utils.cc

```diff
@@ -180,8 +180,6 @@ TEST_CASE("Base64: quick encode / decode with '+', '/' and various paddings", "[
     CHECK(0 == strncmp(encodedMessage, encoded[i], encodedMessageLen));
 
     /* Decode */
-    size_t decodedMessageEstimatedLen = cryptoBase64DecodeSize(encodedMessage, encodedMessageLen);
-    CHECK(strlen(decoded[i]) == decodedMessageEstimatedLen);
```

Comment:
Why was this check deleted?

Reply:
It goes back how `cryptoBase64DecodeSize` determines its size for input. The test here assumes the estimate is not an estimate and rather an accurate value.

here's the run with the check in place (with some extra debugging):
```
experimental/access_control/unit_tests/test_utils.cc:186: FAILED:
  CHECK( strlen(decoded[i]) == decodedMessageEstimatedLen )
with expansion:
  11 == 12
with messages:
  i := 1
  decoded[i] := "ts>ts?ts!!!"

experimental/access_control/unit_tests/test_utils.cc:186: FAILED:
  CHECK( strlen(decoded[i]) == decodedMessageEstimatedLen )
with expansion:
  10 == 12
with messages:
  i := 2
  decoded[i] := "ts>ts?ts!!"
```